### DISCUSSION TITLE
Can Okteto push to an external container registry after building?

### DISCUSSION THREAD

**Discussion by bangbaew** [okteto/okteto#3745] [Q&A]
I've tried this configuration, using Azure storage account, but it turns out to be a physical filesystem like Docker registry server v2 that does not seem to be human controllable (tagging, versioning, getting URI) like Azure Container Registry / Elastic Container Registry / Docker Hub

![image](https://github.com/okteto/okteto/assets/29559559/17dd309e-65c5-4df1-b443-fe974fe556f3)
![image](https://github.com/okteto/okteto/assets/29559559/676cdbe2-9652-4871-aa26-ecc5400c0ffd)

Can I make it push to an external registry like Docker Hub / ECR / ACR, so I can use the image URI in helm charts because I can't make it build from Helm charts, right?
![image](https://github.com/okteto/okteto/assets/29559559/14d605d5-4600-4750-b5c5-5e0ce1575673)

**Comment by pchico83** [comment]
@bangbaew could you post your question in our https://community.okteto.com/ forum? this repo is only for Okteto CLI issues
